h2o-core/src/main/java/hex/grid/HyperSpaceWalker.java

```diff
@@ -437,6 +440,123 @@ public void modelFailed(Model failedModel) {
     }
   } // class CartesianWalker
 
+
+  /**
+   * Hyperparameter space walker which visits each combination of hyperparameters in order.
```

Comment:
In what order? It seems like the order is random.

Reply:
@michalkurka It is a copy paste issue( see CartesianWalker's comment).  PR had WIP label and I had not assigned any reviewers.

Reply:
> PR had WIP label and I had not assigned any reviewers.

No problem! Happy to help and give early feedback!

Reply:
@michalkurka thank you! I appreciate early feedback. Just unfortunately it is difficult at this stage to avoid issues like with this comment.